spec/features/take_whole_pot_calculator_analytics_spec.rb

```diff
@@ -0,0 +1,58 @@
+RSpec.feature 'Take whole pot calculator analytics', type: :feature, js: true do
+  let(:pot) { '100,000' }
+  let(:income) { '7,000' }
+  let(:pension) { '115.95' }
+  let(:pension_invalid) { '500' }
+  let(:pension_frequency) { 'weekly' }
+  let(:pension_frequency_label) { 'Weekly' }
+
+  scenario 'with invalid input' do
+    enter_invalid_data
+    perform_calculation
+
+    expect(data_layer).to include(
+      'pot' => 100_000,
+      'income' => 7_000,
+      'annual_pension' => 26_000,
+      'valid' => false
+    )
+  end
+
+  scenario 'with valid input' do
```

Comment:
A non-JS browser wouldn't be capable of determining the value of `window.dataLayer`?